woocommerce/woocommerce
# Bugfix: Do not repeat rendering the shipping calculator button on the cart page

### Changes proposed in this Pull Request:

When your shipping settings are as follows:

<img width="679" alt="Screenshot 2024-11-08 at 12 14 52 PM" src="https://github.com/user-attachments/assets/39c4ddd7-0b6d-4c4e-a2d8-7ac2335830c1">

The cart page repeats rendering the "button" that shows the shipping calculator:

<img width="537" alt="Screenshot 2024-11-08 at 11 47 05 AM" src="https://github.com/user-attachments/assets/2291bb17-3c72-4656-9a7d-5c4e45501b75">

I believe this might have been introduced in https://github.com/woocommerce/woocommerce/pull/52546 but I haven't confirmed this. This removes rendering of the duplicate button. Afaik this should not affect checkout as the calculator is only for the cart page.


### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. With the shipping settings screenshotted above, and no address provided add items to your cart
2. Visit the cart page
3. See that we render just one link for opening the address panel



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Cart block: Fix a bug where duplicate links are rendered to open the cart shipping calculator.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
